### ISSUE TITLE
Recovering without-systemd.org pages

### ISSUE THREAD

**Issue by clort81** [muellermartin/nosystemd.org#18]
The site had a lot of information which should be made available.

For example the List of Distributions:
https://web.archive.org/web/20190717101155/http://without-systemd.org/wiki/index.php/Linux_distributions_without_systemd

I can set aside some time to curate these here if you give me access.  Nobody seems to be objecting to this idea yet.

**Comment by muellermartin** [comment]
Good idea! Do you know what license these texts where under? A permissive license would make the re-publishing much more feasible, but rewording should be no problem.

Actually I had a link to this wiki, but removed it as it went down (see 4851e65). Maybe we could add a link to the archived version at archive.org.

Feel free to fork this repository and submit pull requests. This way I can review the changes and we can discuss them publicly. I will then happily merge this to fill the page with more information.

Also as the content grows, at some point it would be reasonable to switch to a static page generator (something like Pelican, Jekyll, Hugo or alike) to make editing of multiple pages easier.

**Comment by Obazzi** [comment]
How is this initiative going? Can I help?

**Comment by Obazzi** [comment]
Sadly, I checked everywhere on the archived wiki and I could not find a licence mentioned anywhere. No trace of who made it, so I don't know who to contact to ask for possible permission.

Here is a list of all the possibly useful stuff on the wiki:
https://web.archive.org/web/20190711085730/http://without-systemd.org/wiki/index.php/Arguments_against_systemd
https://web.archive.org/web/20170823100327/http://without-systemd.org/wiki/index.php/Init
https://web.archive.org/web/20190717101155/http://without-systemd.org/wiki/index.php/Linux_distributions_without_systemd
https://web.archive.org/web/20190711085733/http://without-systemd.org/wiki/index.php/Operating_systems_without_systemd
https://web.archive.org/web/20190711085732/https://without-systemd.org/wiki/index.php/List_of_articles_critical_of_systemd
https://web.archive.org/web/20190420093855/http://without-systemd.org/wiki/index.php/Local_copy_of_boycottsystemd.org_archive
Looking at this is really fun! There is a lot of stuff listed that I never even heard about!

For things like the Linux_distributions_without_systemd, Operating_systems_without_systemd, Init and List_of_articles_critical_of_systemd we can just copy selectively what we want from them to put on nosystemd.org without much problem as you can't really claim a list of links or a list of names of operating systems as your own creation anyway. For Arguments_against_systemd we will have to reword.

Also, what do we want the future of nosystemd.org to be? Do we want to make a giant and fun list of a bunch of operating systems like the without-systemd.org wiki was? Or should we refrain to add an OS that even the average Linux nerd would not want to daily drive? For example, this continuation of plan9 called [9front](http://9fs.net/). Should we add this? Is it too esoteric? It is Unix like and has no systemd. If that was not too esoteric, then could we add [Temple OS](https://en.wikipedia.org/wiki/TempleOS) to the list? I am just wondering where is the limit! Just compare the massive list of Linux_distributions_without_systemd of the wiki with the small list that nosystemd.org currently have (some distributions listed in the wiki are probably dead though). And that is without talking about Operating_systems_without_systemd! What is wonderful about nosystemd.org currently is that it is does not overcomplicate the choice of Systemdless OS. A novice can go on nosystemd.org and compare the different OS listed rapidly, and he won't be scared away by a massive list. But maybe we should go all in and start listing a massive amount of OS and distros! And maybe even start putting a small summary next to each one of them! We could probably fix the novice problem by listing all the easy distros first in a smaller, separate section that would come first. But yeah, I want to know everyone's opinion on this.

**Comment by muellermartin** [comment]
Hey @Obazzi thanks for your interest and the input.

I will be honest: I started this project more as a joke than a serious take and I don't really have an idea or vision for it. systemd has matured and it surely has some advantages (for example unit files are really a lot easier to handle). On the other hand, there is room for criticism. Just a plain list of _every_ distro or OS without systemd will not provide any useful help, and other sites like DistroWatch (already linked on nosystemd.org) do a much better job. Working out some solid counterarguments against systemd could be helpful, but I'm not really an expert on init systems and OS design and I lack the time and passion for this. But I will watch the discussion and keep the website running until further notice.
